content/develop/interact/search-and-query/advanced-concepts/vectors.md

```diff
@@ -9,369 +9,455 @@ categories:
+Redis maintains a secondary index on top of your data that has a defined schema. You can include one or more vector fields in your schema using the [`FT.CREATE`]({{< baseurl >}}/commands/ft.create/) command.
 
-    - L2 - Euclidean distance between two vectors
+**Syntax**
 
-         $d(u, v) = \sqrt{ \displaystyle\sum_{i=1}^n{(u_i - v_i)^2}}$
-
-    - IP - Inner product of two vectors
-
-         $d(u, v) = 1 -u\cdot v$
+```
+FT.CREATE <index_name>
+  ON <storage_type>
+  PREFIX 1 <key_prefix>
+  SCHEMA ... <field_name> VECTOR <algorithm> <index_attribute_count>
+    [<index_attribute_name> <index_attribute_value> ...]
+```
 
-    - COSINE - Cosine distance of two vectors
+Refer to the full [indexing]({{< baseurl >}}/develop/interact/search-and-query/indexing/) documentation for additional fields, options, and noted limitations.
 
-         $d(u, v) = 1 -\frac{u \cdot v}{\lVert u \rVert \lVert v  \rVert}$
+**Parameters**
 
-## Create a vector field
+| Parameter | Description |
+|:----------|:------------|
+| `index_name`      | Name of the index.  |
+| `storage_type`    | Storage option (`HASH` or `JSON`).  |
+| `prefix` (optional) | Key prefix that signals to the index which keys should be included. Defaults to all keys if omitted. |
+| `field_name`      | Name of the vector field.  |
+| `algorithm`       | Vector index algorithm (`FLAT` or `HNSW`).  |
+| `index_attribute_count` | Number of vector field attributes.  |
+| `index_attribute_name`  | Vector field attribute name.  |
+| `index_attribute_value` | Vector field attribute value.  |
 
-You can add vector fields to the schema in FT.CREATE using this syntax:
 
-```
-FT.CREATE ... SCHEMA ... {field_name} VECTOR {algorithm} {count} [{attribute_name} {attribute_value} ...]
-```
+### FLAT index
+Choose the FLAT index when you have small datasets (< 1M vectors) or when search accuracy is most important, even at the expense of additional latency.
 
-Where:
+**Required attributes**
 
-* `{algorithm}` must be specified and be a supported vector similarity index algorithm. The supported algorithms are:
+| Attribute          | Description                              |
+|:-------------------|:-----------------------------------------|
+| `TYPE`             | Vector type (`FLOAT16`, `FLOAT32`, `FLOAT64`).  |
+| `DIM`              | Vector dimension. Must be a positive integer; identical for both document and query vectors.  |
+| `DISTANCE_METRIC`  | Distance metric (`L2`, `IP`, `COSINE`).  |
 
-    - FLAT - a simple, straightforward algorithm.
+**Optional attributes**
 
-    - HNSW - Hierarchical Navigable Small World algorithm.
+| Attribute          | Description                                      |
+|:-------------------|:-------------------------------------------------|
+| `INITIAL_CAP`      | Initial vector capacity in the index.            |
+| `BLOCK_SIZE`       | Block size for memory allocation. Defaults to 1024.  |
```

Comment:
Consider dropping these from the docs, as we see it as internal tunning that should not have been exposed in the first place (and we plan to remove it)

Reply:
I will drop them from the documentation!